fhir-model/src/test/java/com/ibm/fhir/model/test/FHIRModelTestBase.java

```diff
@@ -282,113 +262,4 @@ protected JsonObject readJsonObject(String fileName) throws Exception {
-        try {
-            FHIRGenerator.generator( f, false).generate(res, System.out);
-            
-            System.out.println("");
-        } catch (FHIRException e) {
-        	// TODO logging?
-            e.printStackTrace();
-        }
-    }
-
-    protected String getDataAsString(DomainResource res, Format f) {
-        StringWriter writer = new StringWriter();
-        try {
-            FHIRGenerator.generator( f, false).generate(res, writer);
-        } catch (FHIRException e) {
-        	// TODO logging?
-            e.printStackTrace();
-        }
-        return writer.toString();
-    }
-
-    protected String stripNamespaceIfPresentInXML(String str) {
-        String removenameSpace = str.replace(" xmlns:xhtml=\"http://www.w3.org/1999/xhtml\"", "");
-        return removenameSpace.replace("xhtml:", "");
-    }
-
-    protected void runJSONTestForResource(DomainResource res, String filePath) {
-        // DEBUG: Print output to console for manual testing/verification
-        if (DEBUG_JSON) {
-            printOutputToConsole(res, Format.JSON);
-        }
-
-        // Get the generated JSON for Patient resource
-        String jsonActualString = FHIRUtilities.stripNamespaceIfPresentInDiv(getDataAsString(res, Format.JSON));
-
-        // Read the expected JSON for Patient resource from control document
-        String jsonExpectedString = FHIRUtilities.stripNamespaceIfPresentInDiv(readFromFile(filePath));
-
-        // Check if the document matches with the control document and throw exceptions if they differ
-        try {
-            JSONAssert.assertEquals(FHIRUtilities.stripNewLineWhitespaceIfPresentInDiv(jsonExpectedString), FHIRUtilities.stripNewLineWhitespaceIfPresentInDiv(jsonActualString), true);
-        } catch (JSONException e) {
-            e.printStackTrace();
-        }
-    }
-
-    protected void runXMLTestForResource(DomainResource res, String filePath) {
-        // DEBUG: Print output to console for manual testing/verification
-        if (DEBUG_XML) {
-            printOutputToConsole(res, Format.XML);
-        }
-
-        // Get the generated XML for Patient resource
-        String xmlActualString = FHIRUtilities.stripNamespaceIfPresentInDiv(stripNamespaceIfPresentInXML(getDataAsString(res, Format.XML)));
-
-        // Read the expected XML for Patient resource from control document
-        String xmlExpectedString = FHIRUtilities.stripNamespaceIfPresentInDiv(stripNamespaceIfPresentInXML(readFromFile(filePath)));
-
-        // Check if the document matches with the control document and throw exceptions if they differ
-        try {
-            String controlXml = FHIRUtilities.stripNewLineWhitespaceIfPresentInDiv(xmlExpectedString);
-            String testXml = FHIRUtilities.stripNewLineWhitespaceIfPresentInDiv(xmlActualString);
-            
-            // Attribute order is always ignored.
-            Diff diff = DiffBuilder.compare(Input.fromString(controlXml).build())
```

Comment:
do we still use XMLUnit?

Reply:
after deleting various test classes that were commented out, none of these methods were being used and so i deleted them.  I've asked @JohnTimm in the past if we could remove the XMLUnit dependency but he seemed to have preference for keeping it around.  We could revisit that but I'd wanna hear from him.

Reply:
Check.